src/main/java/cs/ualberta/cmput402/tictactoe/board/Board.java

```diff
@@ -93,6 +99,27 @@ else if (isOnLeftDiag(lastColPlayed, lastRowPlayed) && board[0][2].equals(board[
         return false;
     }
 
+    public void checkTie(){
+	if(occupiedSquares == 8){
+	    for (int i = 0; i < 3; i++){
+		for(int j = 0; j < 3; j++){
```

Comment:
Avoid using int literals. 3 should instead be a const variable. Minor change, but should be fixed.

Reply:
As you wish. I'll fix this "the java way". :nauseated_face: